content/src/scripts/containers/Resizer/Resizer.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import "./styles.css";
+import { setWidth } from "../../../../../event/src/actions/UI";
```

Comment:
Fix order of imports. First comes packages and then our local imports and then styles. Each of the sections enter seperated